### PR TITLE
feat: Add audio config to Sectionals.

### PR DESCRIPTION
**Pull request by cmaddox5**
This PR adds the `Audio` config to Sectionals. This will be used by screens to determine if the screen should play audio.

### CODE REVIEW

File: lib/config/v2/busway.ex [hunk 1]
```diff
@@ -11,24 +11,27 @@ defmodule ScreensConfig.V2.Busway do
...
-  alias ScreensConfig.V2.{Departures, EvergreenContentItem}
+  alias ScreensConfig.V2.{Audio, Departures, EvergreenContentItem}
   alias ScreensConfig.V2.Header.CurrentStopName
 
   @type t :: %__MODULE__{
           departures: Departures.t(),
           evergreen_content: list(EvergreenContentItem.t()),
-          header: CurrentStopName.t()
+          header: CurrentStopName.t(),
+          audio: Audio.t()
```
Comment (digitalcora) [inline lib/config/v2/busway.ex:21]: In each case where this field is added, it's in a list that's currently alphabetized. Let's keep it that way?

**Review by digitalcora — COMMENTED** [review]
Curious if this is technically required — afaict the existing v1 Solari config structure does not include any of this customizability, and it seems like most of it would only be meaningful in a context where audio readouts are constantly happening on an interval, rather than on-demand as we have with these screens.

**Comment by cmaddox5** [comment]
> Curious if this is technically required — afaict the existing v1 Solari config structure does not include any of this customizability, and it seems like most of it would only be meaningful in a context where audio readouts are constantly happening on an interval, rather than on-demand as we have with these screens.

As of right now, it is required to get through [this function](https://github.com/mbta/screens/blob/9a045b35cf3de9a27c0d08d0d95b5d16b62fe82e/lib/screens/v2/screen_audio_data.ex#L93). This sort of gets at something I mentioned in the last eng meeting where outside of the three fields called in that function, most of the struct is specific to bus shelter. What we should do is have two separate structs for push button and auto-readout screens. I can tackle that, but I'll make it a separate change.

File: lib/config/v2/busway.ex (hunk 1 above)
Comment (cmaddox5) [reply lib/config/v2/busway.ex:21]: Happy to do that, but I think that might be coincidence. Bus shelter, for example, has it listed last as well. No harm in making the change though.

**Comment by digitalcora** [comment]
Hm, yes, I think we should consider separately (maybe as a group in breakdown) changing how audio is configured so we don't need to set fields that ultimately do nothing.

I guess my main concern would be: After merging this, will the default values for `Audio` ensure that audio always plays, or are we then going to have to manually reconfigure all Sectionals (and remember to configure new ones) with non-default values if we want audio to play?

**Comment by cmaddox5** [comment]
I see what you're saying. So maybe instead of this, Pre Fare, and GL E-Ink having an `audio` field, we just keep a list of push button screen types in the screens code and they just automatically work on deploy?

**Comment by digitalcora** [comment]
That seems reasonable. But also not strictly necessary to sort out as part of this. I'd just want to know if we'd need further manual config changes from here to get Sectional audio to work, once it's implemented in the code. (Pre-Fares seem to use a single "standard" audio config that is not equal to `Audio.from_json(:default)`. Maybe we could specify that here, as the default value for the field in `defstruct`, and then it would automatically populate the first time?)

File: lib/config/v2/gl_eink.ex [hunk 2]
```diff
@@ -23,7 +23,11 @@ defmodule ScreensConfig.V2.GlEink do
             alerts: nil,
             line_map: nil,
             evergreen_content: [],
-            audio: Audio.from_json(:default),
+            audio: %Audio{
+              start_time: ~T[00:00:00],
+              stop_time: ~T[23:59:59],
+              days_active: [1, 2, 3, 4, 5, 6, 7]
+            },
```
Comment (digitalcora) [inline lib/config/v2/gl_eink.ex:30]: I think you meant to add this to `Busway`?

Comment (cmaddox5) [reply lib/config/v2/gl_eink.ex:30]: 🤦

Comment (digitalcora) [reply lib/config/v2/gl_eink.ex]: I suppose we could have a `Audio.always/0` that returns this value, and use that as the default for all the push-button kinds of screens.

**Comment by cmaddox5** [comment]
With this change, we will be able to validate and save the config in place in the admin tool. That will apply the default json we want.

File: lib/config/v2/gl_eink.ex (hunk 2 above)
Comment (cmaddox5) [reply lib/config/v2/gl_eink.ex]: Yeah not a bad idea.

Comment (cmaddox5) [reply lib/config/v2/gl_eink.ex]: Went ahead and changed GL E-Ink and Pre-Fare to use that too. One less step for Ana when she makes more of those screens.

Comment (digitalcora) [reply lib/config/v2/gl_eink.ex]: As long as that won't mess with Permanent Config setup of GL E-ink screens.

Comment (cmaddox5) [reply lib/config/v2/gl_eink.ex]: That tool creates the struct with these values so the default is never used.

**Pull request merged**